src/main/java/org/radarcns/managementportal/Project.java

```diff
@@ -48,8 +48,8 @@
     private final List<Tag> attributes;
     @JsonProperty("projectStatus")
     private final String projectStatus;
-    @JsonProperty("deviceTypes")
-    private final List<DeviceType> deviceTypes;
+    @JsonProperty("sourceTypes")
+    private final List<SourceType> sourceTypes;
 
 
     protected Project(
```

Comment:
Any strict reason to have the fields of this class `final`? If they don't need to be final you can also remove this constructor, reducing the maintenance overhead a little bit.